shared/js/background/wrapper.es6.js

```diff
@@ -9,6 +9,10 @@ export function getExtensionVersion () {
     return manifest.version
 }
 
+export const openExtensionPage = (path) => {
```

Comment:
Nit: As this is a named top-level function, I would prefer to use `function` here instead of an arrow function.